osquery/sql/sqlite_util.cpp

```diff
@@ -27,15 +27,15 @@
 
 namespace osquery {
 
-FLAG(string,
-     disable_tables,
-     "",
-     "Comma-delimited list of table names to be disabled");
-
-FLAG(string,
-     enable_tables,
-     "",
-     "Comma-delimited list of table names to be enabled");
+CLI_FLAG(string,
+         disable_tables,
+         "",
+         "Comma-delimited list of table names to be disabled");
+
+CLI_FLAG(string,
+         enable_tables,
+         "",
+         "Comma-delimited list of table names to be enabled");
```

Comment:
You don't think this should be settable by the local config file? (I guess that could be something for the flag file)

Reply:
Afaik it already doesn't get re-evaluated, so you cannot enable/disable tables at runtime, but most of all, this permits users to disable tables that are affected by CVEs or that are problematic, and they cannot be re-enabled if there's a TLS service that gets hacked.